config/initializers/dogstatsd.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Horizon
+  def self.dogstatsd
+    @dogstatsd ||= Datadog::Statsd.new(
+      Horizon.config[:datadog_trace_agent_hostname],
+      8125,
```

Comment:
@ovasdi - Calling this out since we were confused, but the `datadog` agent does listen for this on port `8125` per the [docs](https://docs.datadoghq.com/developers/dogstatsd/?tab=hostagent#setup).

Reply:
Sorry for throwing you off with some of my suggestions. It seems hooking this up was much simpler in the end.